internal/controller/notificationservice_controller.go

```diff
@@ -76,7 +77,11 @@ func (r *NotificationServiceReconciler) Reconcile(ctx context.Context, req ctrl.
 				logger.Error(err, "Failed to get results", "pipelineRun", pipelineRun.Name)
 				return ctrl.Result{}, err
 			}
-
+			err = r.Notifier.Notify(ctx, string(results))
+			if err != nil {
+				logger.Error(err, "Failed to Notify")
+				return ctrl.Result{}, err
```

Comment:
Maybe we can return a re-queue here for now?

Reply:
requeue added